<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLine</class>
 <widget class="QWidget" name="FLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Polynôme</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="le_solution">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="le_statement">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="le_corr_poly">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Solution</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_polynom"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Correction</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Énoncé</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="le_correction">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btn_correct">
     <property name="text">
      <string>Corriger</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>le_polynom</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FLine</receiver>
   <slot>verify(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le_solution</sender>
   <signal>textChanged(QString)</signal>
   <receiver>FLine</receiver>
   <slot>verify(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_correct</sender>
   <signal>pressed()</signal>
   <receiver>FLine</receiver>
   <slot>show_correct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>verify(QString)</slot>
  <slot>show_correct()</slot>
 </slots>
</ui>
